exporter/faroexporter/config.go

```diff
@@ -0,0 +1,48 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package faroexporter // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/faroexporter"
+
+import (
+	"errors"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/config/confighttp"
+	"go.opentelemetry.io/collector/config/configretry"
+	"go.opentelemetry.io/collector/confmap"
+	"go.opentelemetry.io/collector/exporter/exporterhelper"
+	"go.uber.org/multierr"
+)
+
+// Config defines configuration settings for the Faro exporter.
+type Config struct {
+	confighttp.ClientConfig    `mapstructure:",squash"`
+	exporterhelper.QueueConfig `mapstructure:"sending_queue"`
+	RetryConfig                configretry.BackOffConfig `mapstructure:"retry_on_failure"`
+	FaroEndpoint               string                    `mapstructure:"faro_endpoint"`
```

Comment:
I assume this is leftover from a version before inlining the confighttp.ClientConfig?
```suggestion
```